deployment/terraform/environments/oss-vdb-test/main.tf

```diff
@@ -3,13 +3,15 @@ module "osv_test" {
 
   project_id = "oss-vdb-test"
 
-  public_import_logs_bucket     = "osv-test-public-import-logs"
-  vulnerabilities_export_bucket = "osv-test-vulnerabilities"
-  logs_bucket                   = "osv-test-logs"
-  cve_osv_conversion_bucket     = "osv-test-cve-osv-conversion"
-  debian_osv_conversion_bucket  = "osv-test-debian-osv"
-  backups_bucket                = "osv-test-backup"
-  backups_bucket_retention_days = 5
+  public_import_logs_bucket                      = "osv-test-public-import-logs"
+  vulnerabilities_export_bucket                  = "osv-test-vulnerabilities"
+  logs_bucket                                    = "osv-test-logs"
+  cve_osv_conversion_bucket                      = "osv-test-cve-osv-conversion"
+  debian_osv_conversion_bucket                   = "osv-test-debian-osv"
+  backups_bucket                                 = "osv-test-backup"
+  backups_bucket_retention_days                  = 5
+  affected_commits_backups_bucket                = "osv-test-affected-commits"
```

Comment:
These buckets already exist. Remind me if that going to cause problems @michaelkedar  ?

Reply:
I don't remember if buckets cause problems for these.
I guess we could just try and just `terraform import` if it fails.